model/riscv_types.sail

```diff
@@ -116,11 +116,18 @@ enum Retired = {RETIRE_SUCCESS, RETIRE_FAIL}
 
 /* memory access types */
 
+enum CacheAccessType = {
+  CleanFlush,
+  Inval,
+  Zero,
```

Comment:
Isn't zeroing just a write?

Reply:
It behaves slightly differently because the address of a CBO zero is aligned to the cache block size.